edgelet/Cross.toml

```diff
@@ -3,6 +3,7 @@ passthrough = [
     "BUILD_SOURCEVERSION",
     "VERSION",
     "IOTEDGE_HOMEDIR",
+    "FORCE_NO_UNITTEST",
```

Comment:
Why it is a problem for cross on mac? Could you provide more description?

Reply:
Added description. It is a 100% repro on my Mac.

Reply:
I just remembered that occasionally I've got weird errors due to insufficient memory for docker containers. Once I increase memory limit it works like a charm.

Reply:
I just remembered that occasionally I've got weird errors due to insufficient memory for docker containers. Once I increase memory limit it works like a charm.